service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/RequestStatementMapperTest.java

```diff
@@ -0,0 +1,139 @@
+    private static final String INPUT_JSON_WITH_NOTES = TEST_FILE_DIRECTORY + "example-referral-request-resource-13.json";
+    private static final String OUTPUT_XML_USES_NO_OPTIONAL_FIELDS = TEST_FILE_DIRECTORY + "expected-output-request-statement-1.xml";
+    private static final String OUTPUT_XML_USES_OPTIONAL_FIELDS = TEST_FILE_DIRECTORY + "expected-output-request-statement-2.xml";
+    private static final String OUTPUT_XML_USES_NESTED_COMPONENT = TEST_FILE_DIRECTORY + "expected-output-request-statement-3.xml";
+    private static final String OUTPUT_XML_DOES_NOT_USE_DEFAULT_CODE = TEST_FILE_DIRECTORY + "expected-output-request-statement-4.xml";
+    private static final String OUTPUT_XML_WITH_REASON_CODES = TEST_FILE_DIRECTORY + "expected-output-request-statement-5.xml";
+    private static final String OUTPUT_XML_WITH_SERVICES_REQUESTED = TEST_FILE_DIRECTORY + "expected-output-request-statement-6.xml";
+    private static final String OUTPUT_XML_WITH_DEVICE_REQUESTER = TEST_FILE_DIRECTORY + "expected-output-request-statement-7.xml";
+    private static final String OUTPUT_XML_WITH_ORG_REQUESTER = TEST_FILE_DIRECTORY + "expected-output-request-statement-8.xml";
+    private static final String OUTPUT_XML_WITH_PATIENT_REQUESTER  = TEST_FILE_DIRECTORY + "expected-output-request-statement-9.xml";
+    private static final String OUTPUT_XML_WITH_RELATION_REQUESTER = TEST_FILE_DIRECTORY + "expected-output-request-statement-10.xml";
+    private static final String OUTPUT_XML_WITH_RECIPIENTS = TEST_FILE_DIRECTORY + "expected-output-request-statement-11.xml";
+    private static final String OUTPUT_XML_WITH_RECIPIENTS_AND_PRACTITIONER = TEST_FILE_DIRECTORY
+        + "expected-output-request-statement-12.xml";
+    private static final String OUTPUT_XML_WITH_NOTES = TEST_FILE_DIRECTORY + "expected-output-request-statement-13.xml";
+
+    @Mock
+    private RandomIdGeneratorService randomIdGeneratorService;
+
+    private Bundle bundle;
+    private CharSequence expectedOutputMessage;
+    private RequestStatementMapper requestStatementMapper;
+    private MessageContext messageContext;
+
+    @BeforeAll
+    public static void initialize() {
+        TimeZone.setDefault(TimeZone.getTimeZone(ZoneOffset.UTC));
+    }
+
+    @BeforeEach
+    public void setUp() throws IOException {
+        when(randomIdGeneratorService.createNewId()).thenReturn(TEST_ID);
+
+        var bundleInput = ResourceTestFileUtils.getFileContent(INPUT_JSON_BUNDLE);
+        bundle = new FhirParseService().parseResource(bundleInput, Bundle.class);
+
+        messageContext = new MessageContext(randomIdGeneratorService);
+        messageContext.initialize(bundle);
+        requestStatementMapper = new RequestStatementMapper(messageContext);
+    }
+
+    @AfterEach
+    public void tearDown() {
+        messageContext.resetMessageContext();
+    }
+
+    @AfterAll
+    public static void deinitialize() {
+        TimeZone.setDefault(null);
```

Comment:
We no longer have to use time zoning.

Reply:
NIAD-1060 has not been merged into main yet, so will keep the timezone code for the unit tests for now.

Reply:
Ok